app/tests/spec/views/mixins/password-mixin.js

```diff
@@ -18,7 +18,10 @@ define(function (require, exports, module) {
   var assert = chai.assert;
 
   var PasswordView = BaseView.extend({
-    events: { 'change .show-password': 'onPasswordVisibilityChange' },
+    events: {
```

Comment:
Oooh, the `events` section duplicated in the tests because we created this module before we learned about [Cocktail](https://github.com/onsi/cocktail). By using Cocktail, we can remove the duplicated `events` section.

On line 12, between `var chai` and `var Metrics`, can you add a new line:

```js
var Cocktail = require('cocktail');
```

Delete lines 21-24 (the `events` block).

Replace line 27
```js
_.extend(PasswordView.prototype, PasswordMixin);
```

With
```js
Cocktail.mixin(
  PasswordView,
  PasswordMixin
);
```